e2e-tests/tests/features/dbbackup_pg/20-assert.yaml

```diff
@@ -10,6 +10,22 @@ spec:
   backupStorageName: test-storage
   dbClusterName: test-pg-cluster
 ---
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseClusterBackup
+metadata:
+  name: test-db-backup-azure
+spec:
+  backupStorageName: test-storage-azure
+  dbClusterName: test-pg-cluster
```

Comment:
Why do we need to have these asserts for S3 storage? Why are they created for PG cluster?

Reply:
What do you mean?
This PR is related to GCS and Azure. No changes to S3 have been done.

Reply:
Nevermind, I thought you made changes for asserts only.